src/services/__tests__/tourettes.test.js

```diff
@@ -18,31 +18,49 @@ describe('tourettes', () => {
   describe('sayTourettes', () => {
     describe('when input text is empty', () => {
       const inputText = '';
+      const minAmountOfReplies = 3;
 
       it('replies with at least three items from the profanities list', () => {
         const reply = sayTourettes(inputText);
+        const amountOfReplies = reply.split(' ').length;
         expect(reply).toNotBe(undefined);
-        expect(reply.split(' ').length).toBeGreaterThan(3);
+        expect(amountOfReplies).toBeGreaterThanOrEqualTo(minAmountOfReplies);
+      });
+    });
+
+    describe('when input text is integer', () => {
+      const amountOfProfanities = 25;
+      const inputText = `${amountOfProfanities}`;
+
+      it('replies with a specified amount of profanities', () => {
+        const reply = sayTourettes(inputText);
+        const amountOfReplies = reply.split(' ').length;
+        expect(reply).toNotBe(undefined);
+        expect(amountOfReplies).toBeGreaterThanOrEqualTo(amountOfProfanities);
       });
     });
 
     describe('when inputText is undefined', () => {
       const inputText = undefined;
+      const minAmountOfReplies = 3;
```

Comment:
I'm thinking maybe we want to define this once at the top level, as it has the meaning that _we_ define that there should be at least three replies, I think it looks more formal that it is something that we specify if it's done once at the top level of the file, but feel free to disagree of course :)